magni_description/urdf/magni.urdf.xacro

```diff
@@ -186,7 +186,7 @@
     </xacro:if>
     <xacro:if value="${raspicam_mount == 'downward'}">
       <xacro:raspi_camera name="raspicam" connected_to="base_link">
-        <origin xyz="0.110 0.090 0.455" rpy="3.14159 -0.7854 0.0"/>
+        <origin xyz="0.1 0.090 0.145" rpy="3.14159 -1.25 0.0"/>

```

Comment:
Are these the values for the new bracket?

Reply:
That's what it's intended to be, yes.